packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js

```diff
@@ -27,6 +27,176 @@ const IMPORT_MAPPINGS = {
   }
 };
 
+const indexHtml = `<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>React Spectrum Example</title>
+  </head>
+  <body>
+      <style>
+        body {
+          min-height: 100vh;
+          margin: 0;
+        }
+        #root {
+          min-height: 100vh;
+        }
+        #root > div {
+          padding: 20px;
+          min-height: calc(100vh - 40px);
+        }
+      </style>
+    <div id="root"></div>
+  </body>
+</html>`;
+
+// TODO: Can we use dprint here?
+const formatExampleCode = (code, fixIndention = false) => fixIndention ? code.replace(/`/g, '\\`').replace(/\n/g, '\n    ').trim() : code.replace(/`/g, '\\`').trim();
+
+const formatImports = (imports) => imports.join('\n').replace(/`/g, '\\`');
+
+const getIndexFile = (componentName, colorScheme = 'light') => `import React from "react";
+import ReactDOM from "react-dom";
```

Comment:
I see that these code chunks are what get added to the codesandbox but is there anyway to get rid of the App.js file? Seems like it isn't used

Reply:
> 

Working for me on the built version... still not good to see inconsistencies. I wonder if there were any console warnings or failed network requests.

<img width="983" alt="Screen Shot 2022-03-29 at 9 29 21 AM" src="https://user-images.githubusercontent.com/8961049/160635323-7c6fd9b7-4fa4-4ed0-ae63-87e27b1576f6.png">

Reply:
I tried setting that file to `null` or `undefined`, but was getting errors. Still looking to see if there is a way.

Reply:
might this get rid of it? https://sandpack.codesandbox.io/docs/getting-started/custom-content#custom-entry-file no idea what the ``...`` means, hahaha

Reply:
Thanks for reminding me of this! I didn't see a way to remove it since it is part of the React template we were using, so I just removed our use of the template and used a custom setup. 

While I was there, I switched it to use Parcel instead of CRA, and I upgrade the template to React 18. I also pulled all that string-building into `SandpackFiles.js` as to not clutter the rest of the docs build file.

Reply:
Ended up switching this back to CRA since fragment shorthand wasn't working and I couldn't get a config that made it work.